session/bootstraptest/bootstrap_upgrade_test.go

```diff
@@ -363,6 +363,80 @@ func TestUpgradeVersionForPausedJob(t *testing.T) {
 	require.True(t, suc)
 }
 
+func TestUpgradeVersionForSystemPausedJob(t *testing.T) {
```

Comment:
Shall we add some comment on the case description here?

Reply:
Added.